firedrake/mg/kernels.py

```diff
@@ -334,7 +335,7 @@ def inject_kernel(Vf, Vc):
         %(to_reference)s
         %(evaluate)s
 
-        void inject_kernel(double *R, const double *X, const double *f, const double *Xf)
+        void pyop2_kernel_inject(double *R, const double *X, const double *f, const double *Xf)
```

Comment:
Is it still the case that all of these functions need to start with `pyop2_kernel_`?